src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -235,13 +236,107 @@
    ["SELECT expired FROM certnames WHERE certname=?" certname]
    (comp :expired first sql/result-set-seq)))
 
-(defn purge-deactivated-and-expired-nodes!
+(defn purge-deactivated-and-expired-nodes-old!
```

Comment:
Yeah, that's not even supposed to be in there - was using it for before/after testing.